content/en/docs/application-store/external-apps/deploy-tidb.md

```diff
@@ -0,0 +1,217 @@
+
+   ![add-pingcap-repo](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/add-pingcap-repo.PNG)
+
+3. Your repository displays in the list after successfully imported to KubeSphere.
+
+   ![added-pingcap-repo](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/added-pingcap-repo.PNG)
+
+### Step 3: Deploy the TiDB Operator
+
+1. Log out of KubeSphere and log back in as `project-regular`. In your project, go to **Applications** under **Application Workloads** and click **Deploy New Application**.
+
+   ![deploy-app](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/deploy-app.PNG)
+
+2. In the dialog that appears, select **From App Templates**.
+
+   ![from-app-templates](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/from-app-templates.PNG)
+
+3. Select `pingcap` from the drop-down list, then click **tidb-operator**.
+
+   ![click-tidb-operator](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/click-tidb-operator.PNG)
+
+   {{< notice note >}}
+
+   This tutorial only demonstrates how to deploy TiDB Operator and a TiDB cluster. You can also deploy other tools based on your needs.
+
+   {{</ notice >}}
+
+4. Select the **Chart Files** tab, you can view the configuration from the console directly or download the default `values.yaml` file by clicking the icon on the upper right corner. Under **Versions**, select a version number from the drop-down list and click **Deploy**.
+
+   ![select-version](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/select-version.PNG)
+
+5. On the **Basic Info** page, confirm the app name, app version, and deployment location. Click **Next** to continue.
+
+   ![basic-info](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/basic-info.PNG)
+
+6. On the **App Config** page, you can either edit the `values.yaml` file, or click **Deploy** directly with the default configurations.
+
+   ![check-config-file](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/check-config-file.PNG)
+
+7. Wait for TiDB Operator to be up and running.
+
+   ![tidb-operator-running](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-operator-running.PNG)
+
+8. Go to **Workloads**, you can see two deployments created for TiDB Operator.
+
+   ![tidb-deployment](/images/docs/appstore/external-apps/deploy-tidb-operator-and-cluster/tidb-deployment.PNG)
+
+### Step 4: Deploy a TiDB cluster
+
+The process of deploying a TiDB cluster is similar to deploying the TiDB Operator.
+
+1. Go to **Applications** under **Application Workloads** and click **Deploy New Application** again and then select **From App Templates**.
```

Comment:
Go to **Applications** under **Application Workloads** and click **Deploy New Application** again
->
Go to **Applications** under **Application Workloads**, click **Deploy New Application** again